app/src/main/java/org/tiqr/authenticator/util/databinding/BindingAdapters.kt

```diff
@@ -90,4 +114,43 @@ fun TextView.appName(appName: String) {
 fun View.openBrowser(url: String) {
     if (url.isEmpty()) return
     setOnClickListener(Navigation.createNavigateOnClickListener(MainNavDirections.openBrowser(url)))
+}
+
+/**
+ * Add dividers
+ */
+@BindingAdapter(value = ["dividers"])
```

Comment:
These can be simplified to:

    @BindingAdapter("dividers")

Reply:
I like the label :blush: